lib/config/index.js

```diff
@@ -182,6 +199,9 @@ async function getConfig() {
     config.security.verifyRateLimit = Number(
       await getParameter('security_verify_rate_limit_secs')
     )
+    config.security.callbackRateLimitSeconds = Number(
+      await getOptionalParameter('security_callback_rate_limit_secs')
+    )
```

Comment:
You could potentially avoid the the `Number()` wrap here if you modify `getOptionParameter()` to accept a default value... e.g. `getOptionalParameter = async (id, defaultValue) => ...` then call as `await getOptionalParameter('...', 0)`

Reply:
A default value can't actually be used as a non set value has meaning.

If the config value is not set then only 1 callback is allowed per user (backward compatibility)
If the config value is to set 0 then callbacks are allowed without rate limiting
If the config value is > 0 then callbacks are allowed after that many seconds